copycat-runtime/src/main/java/org/apache/kafka/copycat/cli/Copycat.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package org.apache.kafka.copycat.cli;
```

Comment:
yeah, same namespace is fine.

But right now "copycat --stuff" basically starts a single copycat flow and a single worker.
In the future "copycat --stuff" would do the same, but also submit jobs to the cluster, maybe report on cluster status, etc.

It made sense to me to separate the "run something here and now" and "control cluster" CLIs into two different commands.